.drone.yml

```diff
@@ -14,7 +14,7 @@ steps:
     image: bitnami/kubectl:1.22.2
     user: root # the bitnami/kubectl image has a non-root user set by default, which does not work with the shared docker volume which is owned by root
     commands:
-      - export KUBECONFIG=kubeconfig-$${DRONE_BUILD_STARTED}.yaml
```

Comment:
but I guess, this variable helps to create a unique value filename on the target directory. for example, if this will create something like artifacts... It is very relevant to have. But if these configurations create such loops in exporting this file, then yet yes, it is worth re-configuring but not REMOVING.